config/settings.d/externalipam_netbox.yml.example

```
@@ -0,0 +1,6 @@
+---
+# url is the hostname and path of the Netbox instance
+:url: 'http://netbox-example.com'
```

Slight nit, but I'd use the `example.com` domain and recommend HTTPS.
```suggestion
:url: 'https://netbox.example.com'
```